db/seeds.rb

```diff
@@ -1,9 +1,16 @@
-Rails.logger.info 'Started seeding games'
-Game.states.each_value do |state|
-  10.times { Game.create(state:) }
-end
+# Rails.logger.info 'Started seeding games'
+# Game.states.each_value do |state|
+#   10.times { Game.create(state:) }
+# end
+
+# Rails.logger.info 'Started seeding players'
+# 30.times { Player.create(name: Faker::Name.name) }
```

Comment:
Why comment these?

Reply:
I did not want to seed with another 40 games and 30 players when i seed database with gameplayers.

Reply:
Well, seeds are supposed to be ran at setup stage. If some else would have to clone this project right now, it would be unable to run the seeds, because it has no games or players.

A fix you can do is to change the seeds like
```
Game.states.each_value do |state|
  games_to_create =   Game.where(state: state).count - 10
  games_to_create.times { Game.create(state:) } if games_to_create.positive?
end

(Player.count - 30).times { Player.create(name: Faker::Name.name) }
```

So you make sure you won't create games and players if you have the required count; if you don't, you only create the minimum to reach required count